DXRMapFixups/DeusEx/Classes/DXRFixupM15.uc

```diff
@@ -116,10 +117,13 @@ function PreFirstEntryMapFixes_Bunker()
             break;
         }
     }
-    //Lock the fan entrance top door
+
+    // find the DataLinkTrigger where Page tells you to jump, we use this for finding the door and adjusting its position
     foreach AllActors(class'DataLinkTrigger',dlt){
         if (dlt.datalinkTag=='DL_Bunker_Fan'){ break;}
```

Comment:
probably need to handle the case where this isn't found, just wrap the stuff below in an if none block